.github/DEVELOPMENT.md

```diff
@@ -171,202 +89,6 @@ These are tests that will not run on a device. This is useful for testing device
 │   │   ├── Essentials.UnitTests
 ```
 
-### Reproducing an Issue/Debugging .NET MAUI Code
-Open the .NET MAUI workspace in VSCode.
-In VSCode, select the device that you will be testing on. Using the command palette (ctrl-shift-P/command-shift-P) type `pick device` and
-you will be presented with a set of choices for your target device (Android, iOS, etc). Select one.
-There is a sample project in `src/Controls/samples/Controls.Sample.Sandbox`. This is an empty project
-into which you can add your code to reproduce an issue and also set breakpoints in .NET MAUI source code.
-Let VSCode know this is the project you want to select by going to the command palette (ctrl-shift-P/command-shift-P)
-and typing `pick startup` and select ".NET MAUI: Pick Startup Project" and select the Sandbox project.
-
- Before using the command palette for the first time, you may have to wait a minute
-for intellisense and other tasks to complete before using the command palette. If the project hasn't
-'settled' yet, you will see an error "Pick Startup Project has resulted in an error."
-
-*Note:* When you are committing your PR, do not include your changes to the Sandbox project.
-
-### Integration Tests
-
-The Integration test project under `src/TestUtils/src/Microsoft.Maui.IntegrationTests` contains tests which build and/or run MAUI templates or other projects.
-
-These tests can be ran using the test explorer in VS, or from command line with `dotnet test`. Here's how to run an individual test with parameters from command line:
-
-```bash
-dotnet test src/TestUtils/src/Microsoft.Maui.IntegrationTests --logger "console;verbosity=diagnostic" --filter "Name=Build\(%22maui%22,%22net7.0%22,%22Debug%22,False\)"
-```
-
-You can find detailed information about testing in the [Wiki](https://github.com/dotnet/maui/wiki/Testing).
-
-### Additional Cake Commands
-
-#### Clean
-`--clean`
-- This will do a recursive delete of all your obj/bin folders. This is helpful if for some reason your repository is in a bad state and you don't want to go as scorched earth as `git clean -xdf`
-
-#### Target a specific platform
-`--android`
-`--ios`
-`--windows`
-`--catalyst`
-
-```bash
-dotnet cake --target=VS --workloads=global --android --ios
-```
-
-*Note* you will have to `git clean -xdf` your project if you change or add platforms. 
-
-### Blazor Desktop
-
-To build and run Blazor Desktop samples, check out the [Blazor Desktop](https://github.com/dotnet/maui/wiki/Blazor-Desktop) wiki topic.
-
-### Android
-
-To workaround a performance issue, all `Resource.designer.cs`
-generation is disabled for class libraries in this repo.
-
-If you need to add a new `@(AndroidResource)` value to be used from C#
-code in .NET MAUI:
-
-1. Comment out the `<PropertyGroup>` in `Directory.Build.targets` that
-   sets `$(AndroidGenerateResourceDesigner)` and
-   `$(AndroidUseIntermediateDesignerFile)` to `false`.
```

Comment:
👍  We removed this worklow in .NET 8, but I'm having trouble finding the exact PR.